src/enkf/enkf.f90

```diff
@@ -776,6 +791,8 @@ subroutine enkf_update()
       if (covl_fact < 0.99) print *,'mean covl_fact = ',covl_fact
       if (nskip > 0) print *,nskip,' out of',nobstot,'obs skipped,',nuse,' used'
       if (nsame > 0) print *,nsame,' out of', nobstot-nskip,' same lat/long'
+      if (nuseconvoz > 0 ) print *,nuseconvoz,' out of',nobs_conv + nobs_oz ,'convobs used'
```

Comment:
Why are the conventional and ozone obs lumped together here?  Why not keep them separate as is done elsewhere?

Reply:
They are separated that way in the osense file as the bias predictors are saved for the satellite observations, and the conventional and ozone obs are lumped together in the existing counts (as used in the osense header), so this is what I used for troubleshooting. There's no particular reason why the ozone and conventional observations couldn't be parted out here.

Reply:
Okay that makes sense.  You can leave it.